### DISCUSSION TITLE
Some Questions on Coupling SPPARKS and MyTrim Modules in Magpie

### DISCUSSION THREAD

**Discussion by lvkas521424** [idaholab/moose#27727] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Dear MOOSE and Magpie community maintainers,

Hello, I have a general understanding of the compilation process of MAGPIE by downloading and running it, but I still have many questions. I referred to this question #18986.

1. Are spparks and mytrim co-programmed into libmagpie-opt.so? Instead of them programming their own dynamic link library

2. Where are the .lo and .d files generated by compiling spparks and mytrim used and compiled into dynamic link libraries?

3. Where in the magpie code do they interact with the contents of spparks and mytrim?

lvkas

**Comment by GiudGiud** [comment]
@dschwen @snschune
